keyboards/skdevices/adler/keyboard.json

```diff
@@ -0,0 +1,292 @@
+{
+    "manufacturer": "SKDevices",
+    "keyboard_name": "Adler",
+    "maintainer": "Krzysztof Smosna",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "oled": true
+    },
+    "build":{
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "matrix_pins": {
+        "cols": ["A10", "A9", "A8", "C9", "C8", "C7", "C6", "B15", "B14", "B13", "B12", "B10", "B2", "B1", "B0", "C5", "C4", "A7"],
+        "rows": ["B5", "B4", "B3", "D2", "C12", "C11"]
+    },
+    "processor": "STM32F411",
+    "url": "https://skdevices.pl",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0xA4CE",
+        "vid": "0x0483"
```

Comment:
This is STM's vendor ID, please use a different vendor ID.

Reply:
Sure, I've contacted with STMicroelectronics and I can use that VID with that speciffic PID. This is reserved for that project . Here is reply from support

"" SKDevices“ company can use the original PID: 0xA4CE and VID: 0x0483 pending to be licensed by USB-IF of "STM32F411RET6" and solely for the purpose of Adler using STM32F411RET6 silicon USB device microcontrollers” This license is revocable and nontransferable and no other rights are granted except those expressly stated here.

Please note that They would only be able to test their products for USB-IF certification under our company name. All products will be listed under our company name ( STMicroelectronics) on the Integrators list if certified. They would not have rights to use the USB-IF logos on their products. If they intend to use the USB-IF logos they must obtain their own VID. The requirements to display the USB-IF’s trademarked and licensed USB logo are: 

        1. The company must be a USB vendor (i.e. obtain a USB vendor ID). 

        2. The company must execute the USB-IF Trademark License Agreement. 

        3. The product bearing the logo must successfully pass USB-IF Compliance Testing and appear on the Integrators List under that company’s name. 

These requirements apply to manufacturers and resellers. "